scylla/src/lib.rs

```diff
@@ -246,8 +246,8 @@ pub(crate) mod utils;
 
 /// This module is NOT part of the public API (it is `pub` only for internal use of integration tests).
 /// Future minor releases are free to introduce breaking API changes inside it.
-#[doc(hidden)]
-pub use utils::test_utils;
+#[cfg(test)]
+pub(crate) use utils::test_utils;
```

Comment:
The docstring is outdated - it mentions that module is `pub`.

Reply:
I removed the comment.